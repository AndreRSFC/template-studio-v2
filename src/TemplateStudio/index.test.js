import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { TemplateStudio } from './index';

const templates = [];
const fetchAPTemplates = () => null;
const addNewTemplate = () => null;

const props = {
  templates,
  fetchAPTemplates,
  addNewTemplate,
};

describe('<TemplateStudio />', () => {
  describe('on initialization', () => {
    it('renders page correctly', () => {
      const component = shallow(<TemplateStudio {...props} />);
      const tree = toJson(component);
      expect(tree).toMatchSnapshot();
    });
  });
});
