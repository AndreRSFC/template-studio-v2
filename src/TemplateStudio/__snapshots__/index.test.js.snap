// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TemplateStudio /> on initialization renders page correctly 1`] = `
<div>
  <Header />
  <styled.div>
    <Component
      addTemp={[Function]}
      addToCont={[Function]}
      import={[Function]}
      templates={Array []}
      upload={[Function]}
    />
  </styled.div>
</div>
`;
